sequencing/ATACSeq/jobSubmission/2_batchCalcQCMetrics.sh

```diff
@@ -6,8 +6,8 @@
 #SBATCH --nodes=1 # specify number of nodes.
 #SBATCH --ntasks-per-node=16 # specify number of processors per node
 #SBATCH --mail-type=END # send email at job completion 
-#SBATCH --output=ATACSeq/logFiles/%u/ATACcalcQCS2-%A_%a.log
-#SBATCH --error=ATACSeq/logFiles/%u/ATACcalcQCS2-%A_%a.err
+#SBATCH --output=ATACcalcQCS2-%A_%a.log
+#SBATCH --error=ATACcalcQCS2-%A_%a.err
```

Comment:
On line 52, the mv command needs to change for this to work. Replace 
`./ATACSeq/logFiles/${USER}/`
with
`${SLURM_SUBMIT_DIR}`